LaiWei_HW03.Rmd

```diff
@@ -0,0 +1,49 @@
+---
+title: "LaiWei_HW03"
+author: "Lai"
+date: "7/20/2020"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, error = TRUE)
+```
+
+## Introduction
+
+In this Rmd document, I will practice plotting using the COVID-19 dataset cloned from the repository of New York Times.
+
+```{r load libraries}
+library(ggplot2)
+library(dplyr)
+```
+
+## Reading in data and plotting
+
+In this assignment, I used "us.csv", which contains the COVID-19 case, and death numbers in the US since Jan. 21st, 2020.
+
+```{r, echo=FALSE}
+us_cases = read.csv('us.csv',
```

Comment:
I read my data set in using this same method that you did but when I was looking over Ben's code he was able to achieve the same thing using this line:

states <- read.csv(url("https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-states.csv"))

This allowed him to read in the data directly from the website instead of downloading the dataset like we did so I just thought that was interesting.